src/Type/Type.js

```diff
@@ -0,0 +1,10 @@
+const Type = (props) => {
+
+    return (
+        <>
+            {(props.type != null) ? <span className={'badge rounded-pill bg-primary'}>{props.type}</span> : ''}
```

Comment:
why is it that this component has this check but the Skill component does not have a similar check?